e2e/helpers/testUser.ts

```diff
@@ -62,15 +63,37 @@ export async function cleanupTestUser(testUserId: string, userPage: Page) {
   expect(result).toEqual(testUserId)
 }
 
+async function getSuccessfulLoginResponsePromise(page: Page) {
+  return page.waitForResponse(
+    async response =>
+      response.url().includes('/auth/v1/login2') && response.status() == 201,
+    { timeout: defaultExpectTimeout * 3 }, // allow time for the response to return
+  )
+}
+
+async function getAuthenticatedUserBundleResponsePromise(page: Page) {
+  return page.waitForResponse(
+    async response =>
+      response.url().includes('/repo/v1/user/bundle?mask=127') &&
+      response.status() == 200 &&
+      !(await response.text()).includes('Anonymous'),
+    { timeout: defaultExpectTimeout * 3 }, // allow time for the response to return
+  )
+}
+
+export async function expectAuthenticatedNavDrawerLoaded(page: Page) {
+  await test.step('authenticated nav drawer is loaded', async () => {
+    await expect(page.getByLabel('Search')).toBeVisible()
+    await expect(page.getByLabel('Projects', { exact: true })).toBeVisible()
+    await expect(page.getByLabel('Your Account')).toBeVisible()
+  })
+}
+
 export async function loginTestUser(
   page: Page,
   testUserName: string,
   testUserPassword: string,
 ) {
-  // Perform authentication steps
-  await page.goto('/')
-  await waitForInitialPageLoad(page)
```

Comment:
This page reload is unnecessary -- the page has already navigated to the base URL when this method is called